source/common/upstream/thread_aware_lb_impl.cc

```diff
@@ -185,5 +185,117 @@ LoadBalancerPtr ThreadAwareLoadBalancerBase::LoadBalancerFactoryImpl::create() {
   return lb;
 }
 
+bool ThreadAwareLoadBalancerBase::BoundedLoadHashingLoadBalancer::isHostOverloaded(
+    const Host& host, double weight) const {
+  const uint32_t overall_active = host.cluster().stats().upstream_rq_active_.value();
```

Comment:
We have had multiple bugs in which people disable stats and then features stop working. I think this is one of those case. I think it's very unlikely someone will disable this stat, but at minimum, can you put a TODO above this code that indicates this feature will not work if the rq_active cluster stat is disabled?